app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -21,7 +21,45 @@ keywords:
   - qts
   - Teach First
   - Postgraduate teaching apprenticeship
-
+expander:
+  what-you-need-to-get-onto-the-course:
+    title: what you need to get onto the course
+    text: |-
+      <p>
+      If you have qualifications from outside the UK, you’ll need to show that they meet the standards set for teacher training in England. You can <a href="/non-uk-teachers/non-uk-qualifications">get help comparing English and international qualifications</a>.
+      </p>
+      <p>
+      You’ll need the right to work or study in the UK to do your teacher training in England.</p>
+      <p>
+      If you do not have the right to work in the UK, you should only apply to courses that have visa sponsorship available. You can filter by ‘visa sponsorship’ to <a href="/non-uk-teachers/visas-for-non-uk-trainees">find courses where visas can be sponsored.</a>
+      </p>
+      <p>
+      If your application is successful, the training provider may be able to help you with applying for your visa. <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find out how to apply for your visa to train to teach in England</a>.
```

Comment:
@emmaachesongray This is the wrong link - it's mixed up with line 34

Reply:
changed to the link to find as per live page.